### ISSUE TITLE
Who is going to maintain this repo?

### ISSUE THREAD

**Issue by jdougan** [zenhack/ttrss-sandstorm#47]
As the subject.  I have no idea how to do it myself, though I'm willing to help.

**Comment by ocdtrekkie** [comment]
I have some familiarity with it and rely on it daily, so if I have need to, I can probably take it over. But I've also been planning to do a lot with the WordPress repo for like six months now, so actually setting aside time and doing it is questionable.

**Comment by jdougan** [comment]
And now the tt-rss dev has decided to pack it in. Notice at https://community.tt-rss.org/t/the-end-of-tt-rss-org/7164 .  Archived notice at https://archive.ph/Zjcmo .

The good news is there is  at least one hopeful looking fork at https://github.com/tt-rss/tt-rss

**Comment by ocdtrekkie** [comment]
I grabbed an archival copy of upstream personally as well. I have the app key for this package but haven't had a meaningful reason to try to upgrade it. (I am open to reasons we should.) Fox was pretty aggressive about deprecating things so I wouldn't be surprised to find an update pretty painful.

**Comment by ocdtrekkie** [comment]
Hah, yes, migration would be absolutely brutal: https://community.tt-rss.org/t/support-for-mysql-is-no-more/6897/4

**Comment by jdougan** [comment]
Fox removed the ability to poll feds at non standard port numbers. I had a bunch of workflow that depended on that, I wonder if I can convince the fork to revert that?

**Comment by jdougan** [comment]
For that matter, maybe they can be convinced to put mySQL back in?

**Comment by ocdtrekkie** [comment]
Yeah I think there is a big question on whether the new person will maintain fox's style or become a more approachable project. And tbh a database switch would be well worth it if it was for SQLite, but considering our package already ships two versions of MySQL to handle database migrations, and it looks like a database change requires scripting adding a plugin and probably shipping the entire last release of TTRSS with MySQL support... forever... there's an argument to be made for just greenfielding a new TTRSS package someday (and maybe patching the data migration plugin to this existing one, so people can do the manual jump)
